### PR TITLE
Document pre-commit & code style

### PR DESCRIPTION
**Pull request by mikeage**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,37 @@ See our [Trello](https://trello.com/b/jItetqYe/open-brush) board for more inform
 * [SiMonk0](http://www.furjandesign.com/) for the great new logo!
 * The [SideQuest](https://sidequestvr.com/) team for your support.
 
+## Development / Contributing
```
Comment (mikeskydev) [inline README.md:36]: Should we move this to CONTRIBUTING.md and update it from the template originally provided? There may be a larger overall discussion about what belongs in README and what belongs in CONTRIBUTING

Comment (mikeage) [reply README.md:36]: Sure; do you want me to do this now or was this more of a note for the future?

Comment (mikeage) [reply README.md]: Done (I left the original Contributing there as well for now)

**Review by mikeskydev — APPROVED** [review]

**Comment by billyquith** [comment]
I use [Scoop](https://scoop.sh/) to make Windows tool installation easy. Once installed, dotnet and python can be installed using `scoop install dotnet dotnet-sdk python`.

**Pull request merged**